views/pages/index.ejs

```diff
@@ -13,7 +13,7 @@
     <a href="/" class="lang-logo">
       <img src="/lang-logo.png">
     </a>
-    <h1>Getting Started with Node on Heroku</h1>
+    <h1>Modified text as a title</h1>
```

Comment:
I think that we could change the text here.

Reply:
Cool will change it now